jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessDocumentsServiceImpl.java

```diff
@@ -94,6 +94,8 @@
                                      ex);
                     }
                     documents.add(new DocumentSummary(values[0],
+                                                      serverTemplateId,
+                                                      values[3],
```

Comment:
these two new fields are not needed here. Instead we should change implementation of getDocumentLink method here that will directly build up a complete link with all parameters included.
So this will not be needed any more https://github.com/kiegroup/jbpm-wb/pull/828/files#diff-0cc2a21653eeed1e9b92757c53a3241cR264 as we still should rely on getDocumentLink